internal/controller/controllers/preprovisioningimage_controller_test.go

```diff
@@ -618,7 +618,7 @@ var _ = Describe("PreprovisioningImage reconcile", func() {
 			Expect(res).To(Equal(ctrl.Result{}))
 		})
 
-		It("uses the override ironic image when supplied", func() {
+		It("uses the override ironic image when supplied and does not set the inspector URL", func() {
```

Comment:
I don't think this is always the behavior we want.
Whether the inspector url is provided is dependent on the OCP version of the hub, not on if there is a user override or not.